metaflow/plugins/kfp/kfp.py

```diff
@@ -564,6 +565,53 @@ def _step_cli(self, node: DAGNode, task_id: str, user_code_retries: int) -> str:
         cmds.append(" ".join(entrypoint + top_level + step))
         return " && ".join(cmds)
 
+    @staticmethod
+    def _create_resource_based_node_type_toleration(
+        cpu: float, memory: float
+    ) -> Optional[V1Toleration]:
+        """Allow large enough pod to use higher cost nodes by adding toleration.
+
+        Together with taint added at cluster side,
+        this is a temporary solution to fix "r5.12xlarge host not scaling down" issue,
+        caused by smaller pods keep being scheduled onto large nodes.
+        TODO: Replace using AIP-5264 MutatingWebHook (or Validating) for AIP pod scheduling policies
```

Comment:
can you also add this todo:

`        # TODO(yunw): A Metaflow plugin framework to customize tags, labels, etc.`